internal/rpc/erc721/ethservice_test.go

```diff
@@ -0,0 +1,265 @@
+package erc721
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/freeverseio/laos-universal-node/internal/platform/blockchain"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockRPCClient struct {
```

Comment:
I still don't see the need of using testify, maybe I am missing something but `MockRPCClient` could have been generated via mockgen, couldn't it?